src/test/java/com/microsoft/sqlserver/jdbc/bulkCopy/BulkCopyCSVTest.java

```diff
@@ -72,16 +80,46 @@ static void setUpConnection() {
     @Test
     @DisplayName("Test SQLServerBulkCSVFileRecord")
     void testCSV() {
+        SQLServerBulkCSVFileRecord fileRecord;
+        try {
+            fileRecord = new SQLServerBulkCSVFileRecord(filePath + inputFile, encoding, delimiter, true);
+            testBulkCopyCSV(fileRecord);
+        }
+        catch (SQLServerException e) {
+            fail(e.getMessage());
+        }
+    }
+
+    /**
+     * test simple csv file for bulkcopy by passing a file from url
+     * 
+     * @throws SQLException
+     */
+    @Test
+    @DisplayName("Test SQLServerBulkCSVFileRecord with passing file from url")
+    void testCSVFromURL() throws SQLException {
+        try {
+            InputStream csvFileURL = new URL(
```

Comment:
Suggestion. Instead of `csvFileURL` can you use `csvFileIn` or `csvFileInputStream` or `csvFileStream`